### DISCUSSION TITLE
Proposal: Akash Digest-A Monthly Journal Covering the Akash Ecosystem

### DISCUSSION THREAD

**Discussion by iamGeorgePro** [akash-network/community#356] [Governance Proposals]
**Summary**

Akash-Digest is an important project that creates a monthly Journal about everything happening in the Akash Ecosystem. Its main purpose is to keep the people who care about Akash informed about what's going on and to encourage more people to get involved while also striving to attract new contributors to the project. This monthly digest covers various key areas, including work group and special interest group activities, development progress presented in a simplified, non-technical language, new contributors, governance matters, network health indicators, ecosystem updates, outreach efforts, event summaries, media highlights, market recap, and relevant external news.

Akash-Digest is a detailed monthly report that consolidates and simplifies a wide range of project activities in the Akash Network ecosystem. It is structured to cover key areas of interest, providing insights and updates in a clear and accessible manner. These areas include:

- **Work Groups and Special Interest Groups**: Reports on meetings, activities, and developments within these groups, fostering collaboration and transparency.

- **Development**: Progress within code repositories translated into less technical language to facilitate understanding for a wider audience.

- **People**: Updates on new contributors, community social media statistics, and notable changes in community activity.

- **Governance**: A breakdown of Treasury inflows and outflows, as well as a recap of proposal activities to ensure financial transparency.

- **Network**: Key health indicators for mining, staking, nodes, and research highlights.

- **Ecosystem**: Updates from existing and new services in the Akash ecosystem, showcasing the growth and diversity of the project.

- **Outreach Efforts**: Summarizes the project's outreach initiatives, highlighting efforts to engage with a wider audience.

- **Events**: Provides summaries of both attended and organized events, emphasizing Akash's active involvement in the broader community.

- **Media**: Highlights notable articles, videos, podcasts, and translations related to Akash.

- **Markets Recap**: Provides an overview of Akash's market performance, including trading volume.

- **Relevant External Information**: Includes news related to areas pertinent to Akash, such as decentralization, AI, Cloud, Open Source, governance, funding, chain forks, DEX, and privacy. The focus is on events that provide valuable insights and lessons for the Akash community.

The primary goal of Akash-Digest is to deliver a consolidated monthly report that helps stakeholders track project progress and make informed decisions. The secondary goals are to promote the project by showcasing a significant amount of work each month and attracting new contributors. By adhering to clear production guidelines and using language that is accessible to both technical and non-technical audiences, Akash-Digest aims to convey the depth and quality of the Akash project.

**Objectives**

1. Keep Akash Community Updated: The Akash community is vital to the project's success. By providing them with a monthly digest, we aim to keep them informed about developments and initiatives within Akash.

2. Attract More Builders to the Project: Attracting new contributors and developers is crucial for the project's growth. Akash-Digest serves as a powerful tool to showcase the project's vibrancy and attract skilled individuals.

3. Expose the Quality and Depth of Akash: Akash is a sophisticated platform operating at multiple levels. The digest aims to expose the intricacies and depth of Akash's development, showcasing its advanced features.

4. Convey Akash Culture and Values: The Journal will reflect the values and culture of Akash, emphasizing transparency, community involvement, and innovation.

5. Community Education: Educate the Community about the benefits and functionality of Akash, helping them make informed decisions about using the network.

6. Community is the greatest asset. Highlight community initiatives (projects, articles) and intelligence (discussions).

**Metrics for Success**

- Monthly Readership: Measure the number of readers accessing Akash-Digest each month, providing insights into the reach and engagement of the report.

- Engagement Rate: Evaluate the level of interaction with the report, including comments, shares, and social media mentions.

- New Contributors Attracted: Track the number of new contributors or builders who join the project as a direct result of reading Akash-Digest.

- Community Feedback: Solicit and analyze feedback from the Akash community to continually improve the report's content and format.

**Benefits**

The depth of the information in these publications primarily caters to sophisticated stakeholders, community members, and investors. They can spend about 30 minutes each month to stay up to date with the most important updates and read more if they want to. But don't worry if you're not an expert; the content is still interesting and understandable for everyone. You can pick what you find interesting and skip the parts you don't. Also, the Akash-Digest can be used for different types of updates and sharing information in various ways.

- Transparency: Akash-Digest ensures that all project activities are readily accessible, fostering transparency and trust within the community.

- Knowledge Transfer: The extensive documentation of the production process facilitates knowledge transfer to new maintainers, ensuring Akash digest long-term sustainability.

- Attracting Talent: By showcasing the project's depth, Akash-Digest will attract skilled contributors and builders, enriching the Akash ecosystem.

- Education: The report serves as an educational resource, helping both technical and non-technical audiences understand complex project developments.

- Marketing and Promotion: Akash-Digest promotes the project to a wider audience, showcasing the solid work and progress made each month.

**Budget Breakdown (30K per Year)**

- Content Creation: 50% of the budget (15K) allocated to content creation, including research, writing, editing, and design.

- Publication and Promotion: 20% of the budget (6K) for publishing and promoting content on various platforms like Medium, GitHub, Publish0x, CoinMarketCap, etc.

- Metrics and Analytics: 10% of the budget (3K) for tracking and analyzing metrics to assess the impact of the digest.

- Contingency and Future Planning: 5% of the budget (1.5K) reserved for unforeseen expenses and planning for future expansion or improvements.

**Justification for the Amount**

Content Creation: A significant portion of the budget is dedicated to content creation to ensure that the report provides valuable, in-depth information. High-quality content is essential to meet the project's goals and engage a broad audience.

Promotion and Distribution: Effective promotion is crucial to reach potential builders, contributors, and the wider community. Allocating $6,000 to this category ensures that Akash-Digest is seen and appreciated by the right audience.

Infrastructure and Hosting: Hosting and maintaining the report on multiple platforms enhances its accessibility and resilience. This $2,000 allocation supports the Journals availability across various channels.

Documentation and Knowledge Transfer: Ensuring that the knowledge and processes behind Akash-Digest are well-documented is vital for long-term sustainability. The allocated $2,000 supports the project's ability to transfer knowledge to new maintainers and ensures the report's continuity.

**Comment by Aihonaken** [comment]
Hi! this is a great proposal. Can you elaborate more on the team size, and how the digest will reach people? 
- I know that Akash has its own designer and brand book, will this be included in digest? As the main color and concept? 
-is there an example of such digest, to see how it looks like? 
- who is in charge of proofreading and approving the digest? 
- i know that publishing materials on Medium and github doesn't require any funding, where does the 6k come from? is there a price for some sites to publish? 
- Is there a plan to use the Akash mail base to spread the digest

**Reply by iamGeorgePro** [reply]
Thank you for your kind words and your interest in the proposal. The team behind Akash-Digest primarily consists of myself, @iamGeorgePro (Akash Insider), who will take the lead in creating and managing the digest. While I am the primary contributor, occasional contributions from others are also welcome. To improve resilience and reach, content will be published on Medium, GitHub, Publish0x, Twitter. All sources will be stored in Git repositories that anyone can clone to protect it from loss and censorship. As a bonus it allows offline reading and easier translations. The production process will be extensively documented to facilitate knowledge transfer to new maintainers if necessary.
- Akash-Digest will indeed incorporate the Akash designer and brand book. I aim to maintain a consistent visual identity in line with Akash's branding, including the use of its main colors and branding concepts. I aim to work closely with Special Interest Group-Design to make sure brand guidelines are followed correctly.
- While it's true that platforms like Medium and GitHub offer free publishing options, a portion of the allocated budget is designated for strategic promotional efforts that extend beyond these platforms. These efforts may include features like sponsored posts, social media advertising, collaborations with relevant publications, or even exploring publishing on specialized sites that have a broader reach. By strategically investing in these promotional activities, I aim to maximize the exposure and impact of Akash-Digest, reaching a wider audience and ensuring that the project's updates and insights are widely accessible within and beyond the Akash community.
-  I would love to use the Akash mail base, but I'm not certain about its feasibility at the moment. I plan to conduct further inquiries to determine if it's possible to utilize this resource effectively. If it proves to be viable, I will include it in the proposal when it is ready to go on-chain for voting.

**Reply by Aihonaken** [reply]
thnx for the answer. This questions still remains
- Who is going to be in charge of proofreading
- is there and example of how this digest going to look like.

My only concern is that i just don't see this as a promotional feature for Akash. As an addition to the monthly mail send, this is a strong yes, but as a separate thing to publish on the medium, git, or anywhere else, a guess we have a bunch of validators doing this. HouseofChimera for example.
 
Don't get me wrong, there is just no reason for me to make a paid promotion of the digest.

**Comment by ghost** [comment]
Overall its a nice proposal George! I agree with other comments on Akash discord on the team behind the project, task delegation, community involvement, transparency, funding, and accountability. 

Specially dividing the project into shorter segments for more consistent updates and effort throughout the year

As long as you deliver, I honestly don't care if it's a solo effort or a team effort

**Reply by iamGeorgePro** [reply]
Thank you for your positive feedback and your agreement with the comments from the Akash Discord community. I truly appreciate the support and the constructive insights provided.
- Team and Task Delegation: While I'm the primary contributor, I'm open to occasional contributions from others, and I'm actively exploring opportunities for collaboration to strengthen the project.

- Shorter Segments: The suggestion to divide the project into shorter segments is insightful. I will take this into account and structure the proposal in a way that allows for more consistent updates and effort throughout the year. This will help ensure that the community remains well-informed and engaged. 

- While this proposal covers a full year, I will introduce a quarterly review process. This means that every three months, I'll provide detailed reports to the community. If, after reviewing these reports, it's clear that the project is on the right track and there's a consensus to continue, additional funds will be made available for the project. This approach ensures ongoing transparency and accountability, as it keeps the community regularly updated on the project's developments and allows for collective decision-making regarding funding.

**Comment by piber-dev** [comment]
@instafinanzas already mentioned about that some people have been writing in the insiders chat regarding whether or not it's a one-man effort or a community effort, and some people also want the results to be explained a bit more in-depth. furthermore, if it is a one-man effort, perhaps you can go through other proposals fully or partly regarding e.g., attending events and writing articles. the community budget is currently $5k monthly and there is a sig related to attending events.

**Comment by iamGeorgePro** [comment]
I appreciate the concerns raised regarding whether this project is primarily a one-person effort or a community endeavor. While I am taking the lead on Akash-Digest, I want to emphasize that community involvement is highly valued, and I'm open to contributions from others.
- As for explaining the results in more depth, I absolutely understand the need for comprehensive reporting. I will ensure that the reports provided throughout the project offer detailed insights into our progress, activities, and achievements, giving the community a clear understanding of the results Akash Digest achieving.

- Regarding attending events and writing articles, the current plan is not for me to attend events in person. Instead, Akash Digest aim to collaborate with those who do attend these events. Akash Digest goal is to work with attendees to help them share their experiences, insights, and work with the broader Akash community. By doing so, we can ensure that the community remains well-informed about the activities and experiences of those who participate in events, even if we are not physically present. This collaborative approach allows us to amplify their work and the benefits of these events to the Akash community.

**Comment by CaptainSplince** [comment]
Thanks for taking the inititive and creating this proposal. @iamGeorgePro.

I agree consistent content coming out about Akash is very benificial. However, I believe the Akash Twitter "X" page does a good enough job of updating a community of our size in a consistent manner. 

In addition to the above, my concern is there is no historical precedent for success of this effort. As I've mentioned before, I think allocating resources that are focused on user growth should take priority over any other intitatives. I would want to know exactly how you would correlate user growth to this effort. eg - tracking and reporting.

Final note - I think this idea could potentially be reworked and made into something more community driven (not just you working on) as well as a reduced budget.
